### PR TITLE
Update UserGuide.md for editPatient

### PR DESCRIPTION
**Pull request by LeeMingDe**
Edit user guide to change instructions for editpatient

### CODE REVIEW

**Review by w-yeehong — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -88,22 +88,21 @@ Shows a list of all persons in the address book.
...
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* `NAME` is case-insensitive.
+* `TEMPERATURE` must be to 1 decimal place (e.g. 37.0 instead of 37).
+* `PERIOD_OF_STAY` is in the format YYYYMMDD-YYYYMMDD.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `editpatient john doe p/91234567` Edits the phone number of john doe to be `91234567`.
```
Comment (chiamyunqing) [inline docs/UserGuide.md:105]: Your implementation of editpatient supports editing multiple fields in 1 command. Can consider adding an example of this. E.g. editpatient Alex Yeoh c/Add comment d/20200303-20200315

File: docs/UserGuide.md [hunk 2]
```diff
@@ -173,7 +172,7 @@ Action | Format, Examples
 **Add Patient** | `addpatient n/NAME t/TEMPERATURE d/PERIOD_OF_STAY p/PHONE_NUMBER a/AGE [c/COMMENT]` <br> e.g.,`addpatient n/Betsy Crowe t/36.5 d/20201001-20201014 p/91234567 a/19 c/Is asthmatic`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Edit Patient** | `editpatient NAME [n/NAME] [t/TEMPERATURE] [d/PERIOD_OF_STAY] [p/PHONE_NUMBER] [a/AGE] [c/COMMENT]`<br> e.g.,`edit James Lee t/36.5`
```
Comment (chiamyunqing) [inline docs/UserGuide.md]: In the example, should be editpatient not edit haha ><

**Review by chiamyunqing — CHANGES_REQUESTED** [review]
Minor mistake which I also made

File: docs/UserGuide.md (hunk 2 above)
Comment (LeeMingDe) [reply docs/UserGuide.md]: i thought its already editpatient?

File: docs/UserGuide.md (hunk 1 above)
Comment (LeeMingDe) [reply docs/UserGuide.md:105]: okay can can

**Review by chiamyunqing — APPROVED** [review]
LGTM

**Pull request merged**